### DISCUSSION TITLE
Please use MONAI Discussion tab for questions

### DISCUSSION THREAD

**Discussion by MaoFuyou** [Project-MONAI/MONAI#8105] [General]
**Please use MONAI's Discussions tab**
For questions relating to MONAI usage, please do not create an issue.

Instead, use [MONAI's GitHub Discussions tab](https://github.com/Project-MONAI/MONAI/discussions). This can be found next to Issues and Pull Requests along the top of our repository.
![image](https://github.com/user-attachments/assets/5e11240e-232f-49f4-bb13-7dfa940f5304)
在医学影像生成任务的时候使用 PerceptualLoss 进行计算感知损失，但是在读取预处理文件的时候，不知道为啥跑着跑着代码就远程断掉了。我也尝试了一些方法比如：
![image](https://github.com/user-attachments/assets/90fe8895-c58a-4727-9e61-a74ea327d49f)
但是 依然不好使，我也不知道是否可以开VPN 但是开与关了都试过 还是很不稳定。

**Comment by KumoLiu** [comment]
Hi @MaoFuyou, thanks for your interest here.
Based on the error message, this seems to be a network issue unrelated to the MONAI repository. Please check if you're able to download the pre-trained weights without using MONAI.
Move to discussion for now.

**Reply by MaoFuyou** [reply]
您好，我不太清楚您的意思，这个PerceptualLoss就是从 from monai.losses import PerceptualLoss 导入的啊，然后里面就涉及到了hub等
